defaults/main.yml

```diff
@@ -80,9 +80,12 @@ gitlab_runner_restart_state: restarted
 # default value for force accept self signed certificates
 force_accept_gitlab_server_self_signed: false
 
-# controls diffs for assemle config file
+# controls diffs for assemble config file
 gitlab_runner_show_config_diff: no
 
+# controls logs on ansible configuration tasks, uncomment to prevent secret leaks (Unix support only).
+# gitlab_runner_no_log: yes
```

Comment:
Maybe this name can be a little bit more specific because it doesn't turn off all logs but only a specific category...

Reply:
I don't have a lot of inspiration for this naming… What name should be acceptable for you ?

Reply:
Maybe just: `gitlab_runner_no_log_secrets`

Reply:
Sorry for my answer delay. 

So I pushed a fixup [14ef966](https://github.com/riemers/ansible-gitlab-runner/pull/278/commits/14ef9661a701906804e8c519b41727b17e5a509f) with the new naming: `gitlab_runner_no_log_secrets`.

However, the name still seems misleading to me because it obfuscates more than secrets.

Reply:
@loliee If you are happy with the name (even if it is not 100% accurate but it gives a hint that more or less sensitive information is hidden with that flag), I'm happy to merge.

Reply:
Most important for me is to hide secrets (and avoiding the use of an internal fork)!

So I squashed the fixup.

Thanks you @guenhter for merging this !